LayoutTests/accessibility/shadow-dom/reference-target/mac/aria-describedby.html

```diff
@@ -0,0 +1,34 @@
+
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <script src="../../../../resources/accessibility-helper.js"></script>
+    <script src="../../../../resources/js-test.js"></script>
+</head>
+
+<body>
+<div class="container">
+    <input id="input-1" aria-describedby="x-description-1">
+    <x-description-1 id="x-description-1">
+        <template shadowrootmode="closed" shadowrootreferencetarget="description-1">
+        <span>FAIL IF INCLUDED</span>
+        <span id="description-1">Description 1</span>
+        </template>
+    </x-description-1>
+</div>
+
+<script>
+description("This tests that using aria-describedby to refer to an element indirectly via referenceTarget causes the description to be computed correctly.");
+
+var input1 = accessibilityController.accessibleElementById("input-1");
+let output = expect("input1.customContent", "'description: Description 1'");
+
+debug(output);
+finishJSTest();
```

Comment:
Ditto [my comment in your other PR](https://github.com/WebKit/WebKit/pull/43301#discussion_r2034148897) about not needing this 🙂